packages/opentelemetry-exporter-prometheus/src/prometheus.ts

```diff
@@ -21,19 +21,22 @@ import {
 } from '@opentelemetry/core';
 import {
   CounterSumAggregator,
-  LastValue,
   MetricExporter,
   MetricRecord,
   MetricDescriptor,
-  MetricKind,
   ObserverAggregator,
+  HistogramAggregator,
+  MeasureExactAggregator,
   Sum,
+  Histogram,
+  Distribution,
 } from '@opentelemetry/metrics';
 import * as api from '@opentelemetry/api';
 import { createServer, IncomingMessage, Server, ServerResponse } from 'http';
-import { Counter, Gauge, labelValues, Metric, Registry } from 'prom-client';
+import * as prom from 'prom-client';
 import * as url from 'url';
 import { ExporterConfig } from './export/types';
+import { FakeHistogram, FakeSummary } from './utils';
```

Comment:
Fake doesn't sound really good prefix, I think we should change to something more meaningful.

Reply:
Agree, fake sounds like a test mock

Reply:
I've changed it to `MockHistogram` and `MockSummary` but i'm open to suggestions

Reply:
I'm really not sure what I would consider a good name. Fake and Mock both sound like test mocks to me, but I can't think of anything else. I guess I'm probably fine with either. @mayurkale22 wdyt?

Reply:
These are the few options I can think of:

- `HistogramWrapper`
- `HistogramHandler`
- `CustomHistogram` (I like this one)
- `HistogramRecord`
- `HistogramMetric`
- `HistogramAggregation`

WDYT?

Reply:
I'm fine with `CustomHistogram` and `CustomSummary`

Reply:
@dyladan any thoughts?

Reply:
Custom sounds good to me

Reply:
I modified it to `Custom`